{"ygPerms":{"resourceCapabilityList":[{"resourceType":"GROUP","capabilities":[{"name":"READ"},{"name":"JOIN"}]},{"resourceType":"PHOTO","capabilities":[]},{"resourceType":"FILE","capabilities":[]},{"resourceType":"MEMBER","capabilities":[]},{"resourceType":"LINK","capabilities":[]},{"resourceType":"CALENDAR","capabilities":[]},{"resourceType":"DATABASE","capabilities":[]},{"resourceType":"POLL","capabilities":[]},{"resourceType":"MESSAGE","capabilities":[{"name":"READ"}]},{"resourceType":"PENDING_MESSAGE","capabilities":[]},{"resourceType":"ATTACHMENTS","capabilities":[{"name":"READ"}]},{"resourceType":"PHOTOMATIC_ALBUMS","capabilities":[]},{"resourceType":"MEMBERSHIP_TYPE","capabilities":[]},{"resourceType":"POST","capabilities":[{"name":"READ"}]},{"resourceType":"PIN","capabilities":[]}],"groupUrl":"groups.yahoo.com","intlCode":"us"},"comscore":"pageview_candidate","ygData":{"userId":157334857,"authorName":"kevinmwalters","from":"&quot;kevinmwalters&quot; &lt;kwalters@...&gt;","profile":"kevinmwalters","replyTo":"LIST","senderId":"MkTg_RqooY0lsWJgO6slz91IxdFLzstQRGqjeJCbPnPlh3pvEivfeUAlg2NUYVr_tj_YJyj2hAWV8k1GwYwsy-cxI8a9JeD48mSd0eSg","spamInfo":{"isSpam":false,"reason":"6"},"subject":"Trouble building from cvs on os x 10.4.9","postDate":"1178641027","msgId":1622,"canDelete":false,"contentTrasformed":false,"systemMessage":false,"headers":{"messageIdInHeader":"PGYxcTdxMys5MTV2QGVHcm91cHMuY29tPg=="},"prevInTopic":0,"nextInTopic":1623,"prevInTime":1621,"nextInTime":1623,"topicId":1622,"numMessagesInTopic":3,"msgSnippet":"I m having a little trouble building Cantera 1.7.1 from cvs on my machine (PowerBook G4, ppc, OSX 10.4.9, Python 2.5.1, numarray, gcc, sundials 2.3). Running","rawEmail":"Return-Path: &lt;kwalters@...&gt;\r\nX-Sender: kwalters@...\r\nX-Apparently-To: cantera@yahoogroups.com\r\nReceived: (qmail 94691 invoked from network); 8 May 2007 16:19:07 -0000\r\nReceived: from unknown (66.218.66.70)\n  by m45.grp.scd.yahoo.com with QMQP; 8 May 2007 16:19:07 -0000\r\nReceived: from unknown (HELO n25.bullet.scd.yahoo.com) (66.94.237.54)\n  by mta12.grp.scd.yahoo.com with SMTP; 8 May 2007 16:19:07 -0000\r\nReceived: from [66.218.69.1] by n25.bullet.scd.yahoo.com with NNFMP; 08 May 2007 16:17:07 -0000\r\nReceived: from [66.218.66.89] by t1.bullet.scd.yahoo.com with NNFMP; 08 May 2007 16:17:07 -0000\r\nDate: Tue, 08 May 2007 16:17:07 -0000\r\nTo: cantera@yahoogroups.com\r\nMessage-ID: &lt;f1q7q3+915v@...&gt;\r\nUser-Agent: eGroups-EW/0.82\r\nMIME-Version: 1.0\r\nContent-Type: text/plain; charset=&quot;ISO-8859-1&quot;\r\nContent-Transfer-Encoding: quoted-printable\r\nX-Mailer: Yahoo Groups Message Poster\r\nX-Yahoo-Newman-Property: groups-compose\r\nX-eGroups-Msg-Info: 1:6:0:0\r\nFrom: &quot;kevinmwalters&quot; &lt;kwalters@...&gt;\r\nSubject: Trouble building from cvs on os x 10.4.9\r\nX-Yahoo-Group-Post: member; u=157334857; y=2Hs0fq9cPvRVYkefeRLU61tusuOg5Vc-q38FUmj0z7h20Hk0PDe9hw\r\nX-Yahoo-Profile: kevinmwalters\r\n\r\nI&#39;m having a little trouble building Cantera 1.7.1 from cvs on my\nmachine (=\r\nPowerBook G4, ppc, OSX 10.4.9, Python 2.5.1, numarray, gcc,\nsundials 2.3).\n=\r\n\nRunning the &quot;preconfig&quot; script (unchanged), &quot;make&quot; and &quot;make install&quot;\nseem=\r\ns to go fine without any warnings/errors.  However, when I run\n&quot;make test&quot;,=\r\n or any script that uses importPhase, I get the following\nerror...\n________=\r\n___________________________\n\nProcedure: get_XML_File\nError:   cannot open .=\r\n/diamond.xml for reading.Note, this error\nindicates a possible configuratio=\r\nn problem.\nError opening up file1, diamond.csv\nTraceback (most recent call =\r\nlast):\n  File &quot;frac.py&quot;, line 7, in &lt;module&gt;\n    gas =3D importPhase(&#39;frac.=\r\ncti&#39;)\n  File\n&quot;/Library/Frameworks/Python.framework/Versions/2.5/lib/python2=\r\n.5/site-packages/Cantera/importFromFile.py&quot;,\nline 20, in importPhase\n    re=\r\nturn importPhases(file, [name], loglevel, debug)[0]\n  File\n&quot;/Library/Framew=\r\norks/Python.framework/Versions/2.5/lib/python2.5/site-packages/Cantera/impo=\r\nrtFromFile.py&quot;,\nline 27, in importPhases\n   \ns.append(solution.Solution(src=\r\n=3Dfile,id=3Dnm,loglevel=3Dloglevel,debug=3Ddebug))\n  File\n&quot;/Library/Framew=\r\norks/Python.framework/Versions/2.5/lib/python2.5/site-packages/Cantera/solu=\r\ntion.py&quot;,\nline 39, in __init__\n    preprocess =3D 1, debug =3D debug)\n  Fil=\r\ne\n&quot;/Library/Frameworks/Python.framework/Versions/2.5/lib/python2.5/site-pac=\r\nkages/Cantera/XML.py&quot;,\nline 35, in __init__\n    self._xml_id =3D _cantera.x=\r\nml_get_XML_File(src, debug)\ncantera.error:\n\n*******************************=\r\n*****************\n                Cantera Error!\n**************************=\r\n**********************\n___________________________________\n\nAny ideas how t=\r\no resolve this?  Is preconfig missing something?\n\nThanks,\n\nKevin\n\n\n"}}